adafruit_mpu6050.py

```diff
@@ -76,7 +76,7 @@
 
 
 class Clksel:  # pylint: disable=too-few-public-methods
-    """Allowed values for `_clksel` Register object.
+    """Allowed values for `clock_source` 
```

Comment:
The issue is with `sphinx` need this to be ~annoyingly~ abundantly clear, so you can use :py:attr:`clock_source`.